src/main/java/com/thealgorithms/backtracking/Subsequence.java

```diff
@@ -0,0 +1,53 @@
+package com.thealgorithms.backtracking;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Class generates all subsequences for a given list of elements using backtracking
+ */
+public final class Subsequence {
```

Comment:
What do you think about renaming this class. Perhaps `SubsequenceFinder` would be better.

Reply:
I named it that way to be consistent with other classes like Permutation. 
However, you're right; it would be better to use SubsequenceFinder instead of Subsequence.